docs/source/implementations/clustering.rst

```diff
@@ -13,3 +15,21 @@ https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3167017
 .. automodule:: mlfinlab.clustering.onc
    :members: get_onc_clusters
 
+
+Feature Clusters
+================
+
+.. py:currentmodule:: mlfinlab.clustering.feature_clusters
+.. automodule:: mlfinlab.clustering.feature_clusters
```

Comment:
Add the correct headers:

* Algo description
* Implementation
* Example
* Tutorial Notebook

Please see labelling docs for examples.